README

```diff
@@ -9,29 +9,70 @@ Directory contents:
 		target built from this directory is libest.a.
     /example	This directory contains sample code for implementing
                 an EST client or server.
-    /test	This directory contains unit test code to verify
```

Comment:
Probably would be best to keep this explanation of /test in the public version.  Should be self explanatory, but it's been found to not always be the case.